### PR TITLE
Fix "exception: 'in <string>' requires string as left operand, not type"

### PR DESCRIPTION
**Pull request by CarliJoy**
add also an integration unit test

Fixes https://github.com/sloria/sphinx-issues/issues/125

### CODE REVIEW

**Comment by hugovk** [comment]
Thanks! Confirmed this builds my docs successfully.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```
Comment (sloria) [inline Makefile:1]: Can you remove all the unnecessary files so that the PR only includes the fix?

File: sphinx_issues.py [hunk 2]
```diff
@@ -344,41 +344,39 @@ def setup(app):
         rebuild="html",
         types="[str]",
```
Comment (sloria) [inline sphinx_issues.py:345]: issue: doesn't this need to be changed, too?

```suggestion
        types=[str],
```

File: source/examples.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+This are examples that shall be rendered correctly:
```
Comment (sloria) [inline source/examples.rst:1]: nitpick:

```suggestion
Examples:
```

**Pull request merged**